webui/tests/package.json

```diff
@@ -12,15 +12,17 @@
     "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
     "cypress": "~4.3.0",
+    "cypress-log-to-output": "^1.0.8",
     "eslint": "^6.8.0",
     "eslint-plugin-cypress": "^2.10.3",
     "eslint-plugin-import": "^2.20.2",
-    "cypress-log-to-output": "^1.0.8",
+    "npm-force-resolutions": "^0.0.3",
     "ts-loader": "^6.2.2",
     "typescript": "^3.8.3",
     "webpack": "^4.42.1"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
is this required? looking at package-lock.json without `npm-force-resolutions` seems to not have the vulnerable version.

Reply:
> is this required 🤔 ? looking at package-lock.json without `npm-force-resolutions` seems to not have the vulnerable version.

this is to ensure the `package-lock.json` is patched with what's listed under `resolutions` prior to  an `npm install`. The relevant line from `npm-force-resolutions` package:

> Then add npm-force-resolutions to the preinstall script so that it patches the package-lock file before every npm install you run:

Reply:
gotcha, sorry I misunderstood your point. I've removed it from `webui/tests`.